.github/workflows/main.yml

```diff
@@ -226,20 +226,31 @@ jobs:
       contains(github.event.pull_request.labels.*.name, 'ci:jdk-compatibility')
       || inputs.jdk_compatibility_ci == true
     runs-on: ubuntu-latest
-    env:
-      JAVA_DIST: 'temurin'
     strategy:
+      fail-fast: false #Even if the tests fail on one JDK we want to know if it also fails on others
       matrix:
-        version: [11, 17, 19]
+        include:
+          - version: 11
+            distribution: 'temurin'
+            additionalArgs: ''
+          - version: 17
+            distribution: 'temurin'
+            additionalArgs: ''
+          - version: 20
+            distribution: 'temurin'
+            additionalArgs: ''
+          - version: 21-ea
+            distribution: 'zulu'
+            additionalArgs: '-DsurefireArgLine=-Dnet.bytebuddy.experimental=true'
```

Comment:
Is that still needed now that we have updated to latest bytebuddy version (1.14.4) and also latest ASM (9.5) ?
If it can be removed, then we can also remove the `surefireArgLine` and simplify a bit further.

Reply:
You are right, the upgrade made the `surefireArgLine` part unnecessary. I removed those changes, so that the PR is now nice and small